zstacklib/zstacklib/utils/gpu.py

```
@@ -233,3 +234,18 @@ def reload_hygon_gpu_driver_cmd():
 def is_valid_video_controller(device):
     invalid_keywords = {"iBMC"}
     return all(keyword not in device for keyword in invalid_keywords)
+
+
+def get_vastai_type():
+    r, o, e = bash_roe("lspci | grep Vastai")
```

Ruff (0.12.2)

`bash_roe` may be undefined, or defined from star imports

(F405)